evalml/tests/model_understanding_tests/prediction_explanations_tests/test_explainers.py

```diff
@@ -229,6 +229,7 @@ def test_output_format_checked():
                 Predicted Value: 1
                 Target Value: 2
                 Absolute Difference: 1
+                Index ID: {index_0}
```

Comment:
Do you want to update the documentation to use this new feature?
- https://evalml.alteryx.com/en/latest/user_guide/model_understanding.html#Explaining-Multiple-Predictions

Reply:
@gsheni Can you verify that the index id shows up in the docs built for this branch?

https://evalml.alteryx.com/en/1119-index-id-best-worst/user_guide/model_understanding.html#Explaining-Multiple-Predictions

I'll also add a sentence to the documentation explaining what Index ID refers to!

Reply:
Yep, they show up. Thanks!